deploy/kubernetes/examples/demo-storageclass.yaml

```diff
@@ -15,5 +15,5 @@ parameters:
   csi.storage.k8s.io/controller-expand-secret-namespace: default
 
   csi.storage.k8s.io/fstype: xfs   # Optional. Values ext4\xfs. The default is ext4.
-  volume_name_prefix: demoPVC      # Optional.
+  volume_name_prefix: demo-prefix  # Optional.
```

Comment:
should we use a prefix value that also works "out-of-the-box" for ds8k a well? e.g. just `demo`?

Reply:
I understand what you are saying but it's not self-explanatory, not 'demo' nor 'prefix', and the previous value was not "ds8k friendly" too

Reply:
I agree.
@zingero what do you prefer?

Reply:
I don't understand how ds8k is different in this case.
`demo-prefix` looks good

Reply:
@zingero 
ds8k have a "strong" limit on name length so the prefix has 5 characters limit

Reply:
@zingero [ds8k max prefix length is 5](https://www.ibm.com/docs/en/stg-block-csi-driver/1.6.0?topic=configuration-creating-storageclass)

Reply:
btw, for "config class" yamls, the only way I see to allow different ds8k prefixes is `demo1` and `demo2`

Reply:
I still prefer having a readable and self-explanatory prefix like `demo-prefix`

Reply:
correct me if I'm wrong, but:

1. this is the only value currently not allowing the examples to work "out-of-the-box" with ds8k
2. with my suggestion, the volume name on every storage would be something like `demo_pvc-123...` (which IMO is more natural looking than `demo-prefix_pvc-123`)
3. outside the storage, the value is always seen together with its field name (e.g. `volume_name_prefix: `), which clarifies it

Reply:
if both of you still prefer `demo-prefix`, let me know and I will resolve this

Reply:
@zingero?

Reply:
It's ok to be non "out of the box" since it's a demo example. Our clients probably won't want to use this prefix even without the 5 characters limitation.

Reply:
IMO it is useful for users, especially new users, to be able to easily test the product with minimal configuration changes.
but I'm resolving this.